cmd/osquery-perf/agent.go

```diff
@@ -274,20 +289,168 @@ func (a *agent) runLoop(i int, onlyAlreadyEnrolled bool) {
 	}
 }
 
+func (a *agent) runOrbitLoop() {
+	orbitClient, err := service.NewOrbitClient(
+		"",
```

Comment:
Idea: Could we instead set a tmp dir as `rootDir`? (instead of having to add the `TestNodeKey` thing)

Reply:
that would be ideal, my worry when I was figuring this out is that we might cause problems to the machine running `agent.go` as we'd have a high number of fs writes/reads.

maybe I overreacted?

Reply:
In the future we could use something like an in-memory FS, but... that would require more changes in `OrbitClient`, which would invalidate all our testing :)

So, this is a-ok for now.

Reply:
still might be worth a try? I believe that if we use your suggestion we could get rid of `enrollOrbit` and let the orbit client handle everything, right?

Reply:
Maybe? But you are right, each `agent` will be reading its file every Orbit request.

Reply:
gotcha, seems like the safe way to go is to merge this as-is and revisit soon. Not having the `agent.enrollOrbit` method would be great, the closer we are to the actual implementation the better!

I will create an issue, I also have on-call coming in next week and I will try to dig into this as my side project